src/app.rs

```diff
@@ -25,7 +25,17 @@ pub fn build_app() -> App<'static, 'static> {
                 .long_help(
                     "Include hidden directories and files in the search results (default: \
                          hidden files and directories are skipped). Files and directories are \
-                         considered to be hidden if their name starts with a `.` sign (dot).",
+                         considered to be hidden if their name starts with a `.` sign (dot). \
+                         Flag can be overridden with --no-hidden.",
```

Comment:
Minor: for the long help text, can we please use full sentences ("The flag can be ...").